lambdas/handlers/search_patient_details_handler.py

```diff
@@ -14,32 +15,23 @@
 @set_request_context_for_logging
 @validate_patient_id
 @override_error_check
+@handle_lambda_exceptions
 def lambda_handler(event, context):
     request_context.app_interaction = LoggingAppInteraction.PATIENT_SEARCH.value
+    nhs_number = event["queryStringParameters"]["patientId"]
+    request_context.patient_nhs_no = nhs_number
```

Comment:
Suggest to add an entry point log just for consistency

```suggestion
    request_context.patient_nhs_no = nhs_number
    logger.info("Starting document reference search process")
```